composer.lock

```diff
@@ -1518,29 +1429,28 @@
                 "ssl",
                 "tls"
             ],
-            "time": "2018-04-09T07:21:55+00:00"
+            "time": "2019-01-03T20:21:15+00:00"
         },
         {
             "name": "paragonie/constant_time_encoding",
-            "version": "v1.0.4",
+            "version": "v2.2.3",
```

Comment:
This breaks the PHP 5.6 compatibility. Please restore the previous `composer.lock` and ensure that only `dbunit` is removed (maybe with `composer update dbunit/dbunit`.

Reply:
I think you can add this to `composer.json` to avoid this kind of problem in the future:
```json
  "config": {
    "platform": {
      "php": "5.6.1"
    }
  }
```
(It will make `composer update` ignore any update that is not compatible with PHP 5.6.1.)